### ISSUE TITLE
API users/:user/courses fails when attempting to include deleted enrollments

### ISSUE THREAD

**Issue by fuzzy76** [instructure/canvas-lms#1332]
Set up:
1. Have a user with deleted enrollments
2. Do a GET to users/:user/courses?include[]=term

Result:
500 internal server error

**Comment by fuzzy76** [comment]
Come to think of it, this might not be the optimal place for this report. I won't be offended if you delete it. :)

**Comment by simonista** [comment]
Can you view `/error_reports` and post the error stacktrace?

**Comment by fuzzy76** [comment]
That's the problem. This is an instructure-hosted instance, and I don't seem to have access to those. So I should probably have taken this through those channels?

**Comment by simonista** [comment]
Yeah, if you can submit a ticket through the help menu in the UI, support should be able to help. Keep us posted, 500 can be indicative of a code error.

**Comment by fuzzy76** [comment]
Correction.... `include[]=term` works while `include[0]=term`(which is what PHP creates) doesn't...

**Comment by fuzzy76** [comment]
This issue is still there... As described above.
